lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -119,6 +121,16 @@ class RenderForm extends React.Component {
   }
 
   render() {
+    if (this.state.success) {
```

Comment:
Nit: Would it be possible to refactor this a bit? It seems like there are two flows: success vs. help/contact us. It's all sort of mixed up.

Reply:
yes by default form will be loaded after successful submission success component will be loaded.

Reply:
Are you planning on refactoring this?

Reply:
No I am not sure how you want to refactor this ? Can you suggest something.

Reply:
I would divide this up into two methods: one for the success case and one for the contact-us case. Otherwise, this method is doing two things.